chia/_tests/core/full_node/stores/test_coin_store.py

```diff
@@ -685,6 +696,15 @@ async def test_batch_many_coin_states(db_version: int, cut_off_middle: bool) ->
         assert len(all_coin_states) == (25001 if cut_off_middle else 50000)
 
 
+@pytest.mark.anyio
+async def test_batch_no_puzzle_hashes(db_version: int) -> None:
+    async with DBConnection(db_version) as db_wrapper:
+        # Initialize coin and hint stores.
+        coin_store = await CoinStore.create(db_wrapper)
+        await HintStore.create(db_wrapper)
+        await coin_store.batch_coin_states_by_puzzle_hashes([])
```

Comment:
Yes I can add that - I initially added the test because I found a bug where if the request was empty the query would fail with my old implementation, so I wanted to make sure that was covered in the future.